packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart/ForceFunctionContainerlessStartExecutor.ts

```diff
@@ -51,10 +57,18 @@ export class ForceFunctionContainerlessStartExecutor extends ForceFunctionStartE
     localRun
       .exec()
       .then(msg => {
-        console.log('Gordon resolved: ' + msg);
+        console.log(
+          `localRun resolved in ForceFunctionContainerlessStartExecutor with message: ${msg}`
+        );
       })
       .catch(err => {
-        console.log('Gordon Error: ' + err.message);
+        const errorNotificationMessage = nls.localize(
+          this.UNEXPECTED_ERROR_KEY
+        );
+        telemetryService.sendException(this.UNEXPECTED_ERROR_KEY, err.message);
+        notificationService.showErrorMessage(errorNotificationMessage);
+        channelService.appendLine(errorNotificationMessage);
+        channelService.showChannelOutput();
```

Comment:
Added correct logging and telemetry to the local run executor.  Please let me know if I missed anything.